dts/msm8216-samsung-r05.dts

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "msm8916.dtsi"
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <248 0>;
+	qcom,board-id = <0xF808FF01 5>;
+
+	j53g-eur {
+		model = "Samsung Galaxy J5 2015 (SM-J500H)";
+		compatible = "samsung,j53gxx", "qcom,msm8916", "lk2nd,device";
```

Comment:
I think we should either do

```suggestion
		compatible = "samsung,j53g", "qcom,msm8916", "lk2nd,device";
```

here or

```suggestion
		compatible = "samsung,j53g-eur", "qcom,msm8916", "lk2nd,device";
```

The `xx` is more of an Android convention.

Reply:
I can agree

Reply:
fixed